systems/subscriber/sim-pool/pkg/db/sim_repo_test.go

```diff
@@ -53,8 +54,53 @@ func (u UkamaDbMock) ExecuteInTransaction2(dbOperation func(tx *gorm.DB) *gorm.D
 }
 
 func Test_GetSimsByType(t *testing.T) {
-	t.Run("GetStats", func(t *testing.T) {
+	t.Run("GetSimsByType_Success", func(t *testing.T) {
+		var db *extsql.DB
+
+		db, mock, err := sqlmock.New()
+		assert.NoError(t, err)
+
+		rows := sqlmock.NewRows([]string{"iccid", "msisdn", "is_allocated", "is_failed", "sim_type", "sm_dp_address", "activation_code", "is_physical", "qr_code"})
+		rows.AddRow(
+			"8910300000003540855",
```

Comment:
Lets avoid hard coded values. Lets define them as const in the beginning especially the values that we keep repeatedly using for different testcases. We can hard code one specific values only if we need to change a default value in order to make a specific test pass.

This comment is valid for whole repo.